.github/bin/dist-nuget.sh

```diff
@@ -20,12 +20,14 @@ if [ "$NUGET_API_KEY" = "" ]; then
 fi
 
 # Publish a package only if the repository is upstream (planetarium/libplanet)
-# and the branch is for releases (master or maintenance-*).
+# and the branch is for releases (master or *-maintenance or 9c-*).
 # shellcheck disable=SC2235
-if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || (
-    [ "$GITHUB_REF" = "${GITHUB_REF#refs/tags/}" ] &&
-    [ "$GITHUB_REF" != refs/heads/master ] &&
-    [ "$GITHUB_REF" = "${GITHUB_REF#refs/heads/maintenance-}" ] ); then
+if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || ! [[ \
```

Comment:
How about inversing this condition? It seemed that publishing actual repo is major than dry-run in this case.